### PR TITLE
feat(proposal comments): add comments component, service and integrat…

### PR DESCRIPTION
**Pull request by VERZUOL1**
…ion with api

### CODE REVIEW

**Review by dmytro-almighty — APPROVED** [review]

File: astro_2.0/features/ViewProposal/components/ProposalComments/helpers.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import cn from 'classnames';
```
Comment (AlexeyPonomarenko) [inline astro_2.0/features/ViewProposal/components/ProposalComments/helpers.tsx:1]: This are actually components, not helpers and I consider we should place them in proper files. And do we need them at all?

File: astro_2.0/features/ViewProposal/components/ProposalComments/hooks.ts [hunk 2]
```diff
@@ -0,0 +1,141 @@
+import uniqBy from 'lodash/uniqBy';
```
Comment (AlexeyPonomarenko) [inline astro_2.0/features/ViewProposal/components/ProposalComments/hooks.ts:1]: I consider we should name this file useProposalComments.ts and place it into hooks folder if needed.

File: astro_2.0/features/ViewProposal/components/ProposalComments/helpers.tsx (hunk 1 above)
Comment (VERZUOL1) [reply astro_2.0/features/ViewProposal/components/ProposalComments/helpers.tsx:1]: It is already updated, I removed this file

File: astro_2.0/features/ViewProposal/components/ProposalComments/hooks.ts (hunk 2 above)
Comment (VERZUOL1) [reply astro_2.0/features/ViewProposal/components/ProposalComments/hooks.ts:1]: This hook is not for usage across the application like the hooks we have in hooks folder. It is only relevant for ProposalComments component and placed here just to separate logic an presentation in ProposalComments

**Pull request merged**